spec/queue.js

```diff
@@ -0,0 +1,101 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import Queue from '../src/queue'
+
+chai.use(chaiChange)
+
+describe('Queue', () => {
```

Comment:
`length()` testing may be implied because you're using the `length()` method in some of your other functions, but it's a good idea to increase test coverage by adding a couple of standalone unit tests for `length()`: length of an empty queue and length of a populated queue.